swift/integration-tests/hello-world/test.py

```diff
@@ -0,0 +1,6 @@
+from create_database_utils import *
+
+run_codeql_database_create([
+ 'swift package clean',
+ 'swift build'
+], lang='swift', keep_trap=True)
```

Comment:
the `keep_trap` is just for debugging now? This could be passed as a flag to the test runner, right? Long-term I don't think integration tests should keep traps unconditionally, but I'm ok if it's just to kickoff the tests and development now.

Reply:
Yep. it's a leftover. Cleaning it up right now.

Reply:
@redsun82 done.